src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -462,19 +462,19 @@ class C
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestForSwitchCase_SemanticCheck_AfterLocalConstantVar() =>
-            await VerifyKeywordAsync(AddInsideMethod(@"const object var = null; switch (new object()) { case var $$ }"));
+            await VerifyAbsenceAsync(AddInsideMethod(@"const object var = null; switch (new object()) { case var $$ }"));
```

Comment:
i specifically regressed this behavior because:
1. teh logic to support it was such a pain to keep
2. supporting users explicitly namign things like `var` is explicitly carved out as something we do not cater for in the IDE.